cumulus_etl/upload_notes/cli.py

```diff
@@ -53,6 +54,15 @@ async def gather_docrefs(
         )
 
 
+def datetime_from_docref(docref: dict) -> datetime.datetime | None:
+    """Returns the date of a docref - preferring `context.period.start`, then `date`"""
```

Comment:
This is preferring clinically-relevant time over the time-of-DocRef creation. That feels right to me, but I'm open to that being flipped.

Reply:
I think that is probably the right call.